content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -38,10 +38,9 @@ IPv4/IPv6 dual-stack on your Kubernetes cluster provides the following features:
 
 The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack Kubernetes clusters:
 
-   * Kubernetes 1.20 or later  
-     For information about using dual-stack services with earlier
-     Kubernetes versions, refer to the documentation for that version
-     of Kubernetes.
+   * Kubernetes v{{< skew currentVersion >}}.
++    If you are not running Kubernetes v{{< skew currentVersion >}}, check the documentation for
++    that version of Kubernetes. {{< version-check >}}
    * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
    * A network plugin that supports dual-stack (such as Kubenet or Calico)
```

Comment:
This needs a bigger reword I'm afraid to say. We shouldn't mention kubenet or calico. We can rely on the earlier `
{{< feature-state for_k8s_version="v1.23" state="stable" >}}` shortcode to explain what people need there.

One phrase that might work for part of that
```suggestion
A version of Kubernetes that supports dual-stack networking. Dual-stack networking is generally available
since Kubernetes v1.23, so if you are running Kubernetes v{{< skew currentVersion >}} then this
feature is available to you.
If you are not running Kubernetes v{{< skew currentVersion >}}, check the documentation for
that version of Kubernetes. {{< version-check >}}

Your cluster must also meet the following requirements:
```

Reply:
Suggested change on line 39 does not looks right ,please take a look🙂

Reply:
@PriyanshuAhlawat : I find the suggested change to be okay. Could you please be more specific as to what, in particular, is the problem with the rephrasing that @sftim has suggested?